### PR TITLE
[testing] tree: promote changes from testing-devel at 085e57099ef251357388beab6…

### PR DESCRIPTION
**Pull request by lucab**
…452981b8ece2389

### CODE REVIEW

File: manifest-lock.x86_64.json [hunk 1]
```diff
@@ -1096,52 +1108,52 @@
...
     "sssd-ldap": {
-      "evra": "2.4.0-2.fc33.x86_64"
+      "evra": "2.4.0-4.fc33.x86_64"
     },
     "sudo": {
       "evra": "1.9.2-1.fc33.x86_64"
     },
     "systemd": {
-      "evra": "246.7-2.fc33.x86_64"
+      "evra": "246.7-1.fc33.x86_64"
```
Comment (travier) [inline manifest-lock.x86_64.json:1141]: Is this expected?

File: fedora-rawhide.repo [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        https://dl.fedoraproject.org/pub/fedora-secondary/development/rawhide/Everything/$basearch/os/
+#metalink=https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=$basearch
+enabled=1
+countme=1
+metadata_expire=6h
+repo_gpgcheck=0
+type=rpm
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
+skip_if_unavailable=False
```
Comment (travier) [inline fedora-rawhide.repo:16]: Is this expected?

**Review by travier — COMMENTED** [review]
The systemd downgrade looks suspicious.

File: manifest-lock.x86_64.json (hunk 1 above)
Comment (lucab) [reply manifest-lock.x86_64.json:1141]: It happened in https://github.com/coreos/fedora-coreos-config/commit/727aa236e51fca47f9d2579dc24794169708c98a.

File: fedora-rawhide.repo (hunk 2 above)
Comment (lucab) [reply fedora-rawhide.repo:16]: I think so, based on https://github.com/coreos/fedora-coreos-config/pull/792.

/cc @jlebon @dustymabe

File: manifest-lock.x86_64.json (hunk 1 above)
Comment (dustymabe) [reply manifest-lock.x86_64.json:1141]: Yeah. Let me explain.

This was basically the generated lockfile getting back in line with the overrides. I set the systemd override in 969380b and the next time the lockfile bumper ran (727aa23) it dropped it back down to `246.7-1`.

File: fedora-rawhide.repo (hunk 2 above)
Comment (dustymabe) [reply fedora-rawhide.repo:16]: Yes, expected. The repo exists but is unused (usage is controlled by the entries in `manifest.yaml`).

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by lucab** [comment]
@dustymabe from your replies to @travier this is ok as-is, right?

**Review by jlebon — APPROVED** [review]

**Review by travier — APPROVED** [review]

**Pull request merged**